Resources/public/js/services/ez-googlemapapiloader.js

```diff
@@ -69,8 +69,9 @@ YUI.add('ez-googlemapapiloader', function (Y) {
          * `mapAPIReady` or `mapAPIFailed` depending on results
          *
          * @method load
+         * @param String key to use for api calls, can be omitted but then map might not work
          */
-        load: function () {
+        load: function ( key ) {
```

Comment:
(same in ez-maplocation-editview.js and ez-maplocation-view.js and in -tests.js)